app/javascript/react/screens/App/Overview/components/ModalWizard/ModalWizard.js

```diff
@@ -2,7 +2,14 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { noop, Modal, Wizard, Icon, Button } from 'patternfly-react';
 import { connect } from 'react-redux';
-import { find } from 'lodash';
+
+const reduxFormMap = {
+  'Infrastructure Mapping Wizard': [
```

Comment:
i think we just construct it using __('Infrastructure Mapping Wizard') so it lines up i guess...

nice job on this  fix...